ecosystem/resources/members/electronic-structure-orbital-optimization.toml

```
@@ -0,0 +1,14 @@
+name = "electronic-structure-orbital-optimization"
+url = "https://github.com/JoelHBierman/electronic-structure-orbital-optimization"
+description = "This repository contains a collection of methods which apply orbital optimization to quantum algorithms for finding ground and excited states of the electronic structure Hamiltonian. These methods optimize the underlying basis set through orbital rotations, enabling more accurate energies to be found while using as few qubits as possible."
```

@JoelHBierman I've tried to rephrase to keep it under 30 words and using more actionable language. Is this still accurate?

```suggestion
description = "Apply orbital optimization to quantum algorithms that solve electronic structure Hamiltonians. This collection of methods optimize the underlying basis set through orbital rotations, finding more accurate energies with fewer qubits."
```

---

Yes, that looks good to me.